### PR TITLE
fixes #50

### LINKED ISSUE #50: Improve the API for File objects
It took me way too long to find out how to get the contents of a file as a string (`.contents.toString('utf8')`). It would be *really* helpful to have this in the readme.

Thanks!

### PR DESCRIPTION
**Pull request by andyperlitch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,15 @@ Returns the stream.
 
 Returns a pretty String interpretation of the File. Useful for console.log.
 
+### contents
+
+The [stream](https://nodejs.org/api/stream.html#stream_stream) or [buffer](https://nodejs.org/api/buffer.html#buffer_class_buffer) of the file. For example:
+```js
+if (file.isBuffer()) {
+    var contents = file.contents.toString('utf8')
```
Comment (contra) [inline README.md]: var inside an if is bad JS

Comment (phated) [inline README.md]: @contra no it isn't and it's actually correct as per node styleguide

Comment (contra) [inline README.md]: @phated guess i got brainwashed by crockford

Comment (andyperlitch) [inline README.md]: lol i was actually thinking about this while writing the example but then i was like mehhhh adding a var declaration above seemed like too much...

Comment (sindresorhus) [inline README.md]: The `.toString` argument is also moot as it's the default.

**Comment by stevemao** [comment]
This is fixed on the master

**Comment by contra** [comment]
@stevemao guess GH doesn't close multiple issues per commit msg

**Comment by stevemao** [comment]
No, https://help.github.com/articles/closing-issues-via-commit-messages/
You have to say `closes #34, closes #23`

**Pull request closed without merging**